catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/model/Field.js

```diff
@@ -134,6 +134,10 @@ define([
                     if(this.get('valueUpperLon') && (this.get('valueUpperLon') > 180 || this.get('valueUpperLon')< -180)){
                         error = 'Invalid decimal longitude value. Must be -180 < lat < 180';
                     }
+                } else if(type === 'date'){
+                    if (isNaN(new Date(this.get('valueDate') + 'T' + this.get('valueTime') + 'Z'))) {
```

Comment:
actually, I think `_.isDate(new Date(this.get('valueDate') + 'T' + this.get('valueTime') + 'Z')).getTime()` will work.